### PR TITLE
add audience to custom_service_account

### PR DESCRIPTION
**Pull request by dacozai**
Fix: #99 

## Summary
Adding Audience into Custom Service Account to support specifying Audience function.

### CODE REVIEW

File: src/custom_service_account.rs [hunk 1]
```diff
@@ -167,9 +175,14 @@ impl<'a> Claims<'a> {
...
         let iat = Utc::now().timestamp();
+        let aud = if let Some(aud_str) = aud {
+            aud_str
+        } else {
+            &key.token_uri
+        };
         Claims {
             iss: &key.client_email,
-            aud: &key.token_uri,
+            aud,
```
Comment (djc) [inline src/custom_service_account.rs:185]: Would prefer to inline directly here:

```rust
aud: aud.unwrap_or(&key.token_uri),
```

Comment (dacozai) [reply src/custom_service_account.rs]: Thank you 🙏🏻 Here is the update [347cdfd](https://github.com/djc/gcp_auth/pull/119/commits/347cdfd7f2c111b1569a471a60feaa887a09c5c7)

**Review by djc — APPROVED** [review]

**Comment by djc** [comment]
If you'd like a release, please submit a PR with a version bump.

**Comment by dacozai** [comment]
> If you'd like a release, please submit a PR with a version bump.

No problem. Thank you 🙏 😊

**Pull request merged**